manim/mobject/geometry.py

```diff
@@ -912,6 +912,28 @@ def __init__(self, *args, **kwargs):
 
 
 class CubicBezier(VMobject):
+    """
+    Example
+    -------
+
+    .. manim:: BezierSplineExample
+        :save_last_frame:
+
+        class BezierSplineExample(Scene):
+            def construct(self):
+                p1 = np.array([-3, 1, 0])
+                p1b = p1 + [1, 0, 0]
+                d1 = Dot(point=p1).set_color(BLUE)
+                l1 = Line(p1, p1b)
+                p2 = np.array([3, -1, 0])
+                p2b = p2 - [1, 0, 0]
+                d2 = Dot(point=p2).set_color(RED)
+                l2 = Line(p2, p2b)
+                bezier=CubicBezier([p1b, p1b + 3 * RIGHT, p2b - 3 * RIGHT, p2b])
+                self.add(l1, d1, l2, d2, bezier)
```

Comment:
Ran black formatting against the code:
```suggestion
        class BezierSplineExample(Scene):
            def construct(self):
                p1 = np.array([-3, 1, 0])
                p1b = p1 + [1, 0, 0]
                d1 = Dot(point=p1).set_color(BLUE)
                l1 = Line(p1, p1b)
                p2 = np.array([3, -1, 0])
                p2b = p2 - [1, 0, 0]
                d2 = Dot(point=p2).set_color(RED)
                l2 = Line(p2, p2b)
                bezier = CubicBezier([p1b, p1b + 3 * RIGHT, p2b - 3 * RIGHT, p2b])
                self.add(l1, d1, l2, d2, bezier)
```